templates/cert-manager.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: cert-manager
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: cert-manager
+  annotations:
+    appversion.kubeaddons.mesosphere.io/cert-manager: "0.1.0"
+    docs.kubeaddons.mesosphere.io/cert-manager: "https://docs.cert-manager.io/en/release-0.10/"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.7.0
```

while the _chart_ may support this low of a version Addons in kubeaddons general take the stance of v1.15+ as per [kubeaddons requirements](https://github.com/mesosphere/kubeaddons#requirements) and just out of general caution. Any reason we shouldn't update this?
```suggestion
    minSupportedVersion: v1.15.0
```